cli/ast/mod.rs

```diff
@@ -43,6 +43,8 @@ use swc_ecmascript::transforms::react;
 use swc_ecmascript::transforms::typescript;
```

Comment:
Note that I moved this file... hopefully that's ok, but adding the transforms iside the `ast.rs` file would have started to get really busy.

Reply:
Looks fine for me 👍